qa/tasks/ses_qa.py

```diff
@@ -142,6 +144,36 @@ def begin(self):
                 raise ConfigError(self.err_prefix + "No such method ->{}<-"
                                   .format(method_spec))
 
+    def drive_replace_initiate(self, **kwargs):
+        """
+        Initiate Deepsea drive replacement
+
+        Assumes there is 1 drive not being deployed (1node5disks - with DriveGroup `limit: 4`)
+
+        In order to "hide" an existing disk from the ceph.c_v in teuthology
+        the disk is formatted and mounted.
+        """
+        # :TODO: make this random and find correct disk device
+        osd_id = "2"
+        total_osds = self.master_remote.sh(number_of_osds_in_cluster)
+        assert int(total_osds) == 4
```

Comment:
`assert int(total_osds) == 4, "Unexpected number of osds {} (expected 4)".format(total_osds)`